apiary.apib

```diff
@@ -332,24 +332,32 @@ Get the current authenticated user.
             + last_name: `Smith` (string)
             + email: `john@teamleader.eu` (string)
             + telephones (array[Telephone])
-            + language: `nl-BE` (enum[string])
-                + Members
-                    + da-DK
-                    + de-DE
-                    + en-GB
-                    + es-ES
-                    + fi-FI
-                    + fr-FR
-                    + it-IT
-                    + nb-NO
-                    + nl-BE
-                    + nl-NL
-                    + pl-PL
-                    + pt-PT
-                    + sv-SE
-                    + tr-TR
             + function: `Sales` (string)
-            + time_zone: `Europe/Brussels` (string)
+            + preferences (object)
+                + language: `nl-BE` (enum[string])
+                    + Members
+                        + da-DK
+                        + de-DE
+                        + en-GB
+                        + es-ES
+                        + fi-FI
+                        + fr-FR
+                        + it-IT
+                        + nb-NO
+                        + nl-BE
+                        + nl-NL
+                        + pl-PL
+                        + pt-PT
+                        + sv-SE
+                        + tr-TR
+                + time_zone: `Europe/Brussels` (string)
+                + date_format (object)
+                    + short: `dd/mm/yyyy` (string)
+                    + medium: `dd mmm yyyy` (string)
+                    + long: `dd mmmm yyyy` (string)
+                + number_format (object)
```

Comment:
Q: Could we instead have a preference for region, instead of `date_format` and `number_format`? Not sure it's worth having such granularity, my guess would be that users probably just want to set it to "Germany".

Reply:
I like the granular approach more because it's more explicit and flexible. I'm guessing there could be a lot of possible regions, with its own preferences. An integration would then also need the knowledge about what regions need what formatting, the granular approach is self-documenting.

Reply:
@regularjack, thing is that many front services will need to still translate that region to a valid date and number format. Either we need to have all of these different rules in every service and share that with every third party service to have something consistent, or we provide it in a clean way so that there's no debate on the format.